schema/flavor.json

```diff
@@ -24,7 +24,8 @@
             "type": "string",
             "enum": [
                 "IND-CPA",
-                "IND-CCA"
+                "IND-CCA",
```

Comment:
This is a signature scheme, how can it be IND-CCA secure?

Reply:
No, the line I added is the EUF-CMA one. (I hate JSON for not allowing trailing commas grml)

Reply:
this shouldn't be necessary anymore (and not in this patch anyway :slightly_smiling_face: )